SimG4Core/PhysicsLists/src/EmParticleList.cc

```diff
@@ -1,24 +1,26 @@
-#ifndef SimG4Core_Application_CMSEmParticleList_h
-#define SimG4Core_Application_CMSEmParticleList_h 1
+#include "SimG4Core/PhysicsLists/interface/EmParticleList.h"
 
-// V.Ivanchenko 6 March 2017
-// List of Geant4 basic particle names used in SIM step 
-
-#include "globals.hh"
-
-static const G4int nEmParticles = 39;
-static const G4String EmPartNames[nEmParticles] = 
-{ 
+EmParticleList::EmParticleList() 
```

Comment:
@davidlange6 , not work for me out of the box, however, current code is simpler, no static, string hidden from the rest of the sub-library. A vector is used instead of array to easy use c++11 loop.